packages/archway-proto/package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@archwayhq/archway-proto",
+  "version": "0.0.1",
+  "description": "Typescript protobuf client generator",
+  "author": "David Choi <david.choi@philabs.xyz>",
+  "homepage": "https://github.com/jsoneaday/archway-proto#readme",
+  "license": "Apache-2.0",
+  "main": "main/index.js",
+  "module": "module/index.js",
+  "typings": "types/index.d.ts",
+  "directories": {
+    "lib": "src",
+    "test": "__tests__"
+  },
+  "files": [
+    "types",
+    "src",
+    "main",
+    "module"
+  ],
```

Comment:
Please, check the `arch3-core` package metadata, and let's keep the same info on both.